jquery.simplebanner.js

```diff
@@ -140,6 +141,10 @@
         self.element.find('.bannerList').stop(false,true).animate({
             'marginLeft':-slideIndex * self._bannerWidth
         },self.options.animTime);
+        
+        if(self.options.slideChange !== null){
```

Comment:
Should rather check if it's a function, in case it's erroneously set to an object/boolean/etc.